system/shaders/GL/1.5/gl_tonemap.glsl

```diff
@@ -1,4 +1,50 @@
-float tonemap(float val)
+#if (defined(KODI_TONE_MAPPING_REINHARD) || defined(KODI_TONE_MAPPING_ACES) || defined(KODI_TONE_MAPPING_HABLE))
```

Comment:
These constants only are used by ACES and HABLE --> `defined(KODI_TONE_MAPPING_REINHARD)` can be removed